src/webview/css/css.js

```diff
@@ -257,8 +257,8 @@ hr {
 .message_inline_ref img,
 .twitter-image img {
   width: 100%;
-  height: 30vh;
-  object-fit: contain;
+  height: 200px;
+  object-fit: cover;
```

Comment:
I don't think we can just switch to `cover` here. In many cases (wide landscape images) it cuts out the side parts of the image, which in some cases might be OK, but in others result in a weird zoom. Maybe we need more logic around this. If it would cut more than 20-50% of the image do not do this...